cockroachcloud/authorization.md

```diff
@@ -35,66 +35,119 @@ For the main pages covering users and roles at the SQL level within a specific d
 
 ## Organization user roles
 
-When a user is first added to an organization, they are granted the default role, **Org Member**, which grants no permissions and just indicates the membership in the organization. Org. or Cluster Administrators may edit the roles assigned to organization users in the {{ site.data.products.db }} console's **Access Management** page, or using the {{ site.data.products.db }} API / Terraform Provider.
+When a user is first added to an organization, they are granted the default role, **Org Member**, which grants no permissions and only indicates membership in the organization. Org or Cluster Administrators may edit the roles assigned to organization users in the {{ site.data.products.db }} console's **Access Management** page, or using the {{ site.data.products.db }} API / Terraform Provider.
 
 To learn more, refer to [Manage organization users](managing-access.html#manage-an-organizations-users)
 
 The following roles may be granted to {{ site.data.products.db }} organization users within a specific organization:
 
-- Org Member
-- Org Administrator (legacy)
-- Org Developer (legacy)
-- Cluster Developer
-- Cluster Admin
-- Cluster Creator
-
-### Organization member
-
-This default role is granted to all organization users once they are invited. It grants no permissions to perform cluster or org actions.
-
 ### Org Administrator (legacy)
 
 Org Administrators can manage the organization and its members, clusters, and configuration. This role grants the user permissions to perform all critical functions managing a {{ site.data.products.db }} organization:
 
-- [Create a cluster](create-your-cluster.html)
+- [Create or delete a cluster](create-your-cluster.html)
 - [Invite Team Members to the organization](managing-access.html#invite-team-members-to-an-organization)
 - [Manage an organization's users and their roles](managing-access.html#manage-an-organizations-users)
 - [Create and manage SQL users](managing-access.html#create-a-sql-user)
 - [Manage billing for the organization](billing-management.html)
 - [Restore databases and tables from a {{ site.data.products.db }} backup](use-managed-service-backups.html#ways-to-restore-data)
 - [Delete an organization](managing-access.html#delete-an-organization)
+
 {{site.data.alerts.callout_info}}
-In a future release, this role will be deprecated in favor of more fine-grained roles for separately administering organization-level user-management functions, cluster management functions, and billing management functions.
+This role is deprecated in favor of the following more fine-grained roles, which, in combination, cover the same permissions:
```

Comment:
@MichaelTrestman @matthewtodd @pawalt - Should we indicate that it would be "deprecated soon" when the feature is GA, rather than mention the deprecation now? Asking because we're sending that GA and deprecation notification to all customers for the week of 7/10.